scripts/release/mac-remote-build

```
@@ -90,4 +90,5 @@ EOF
 echo '=======>>>> Retrieving mac compiled binaries'
 rsync -avz --ignore-existing ${remote_macos_host}:"${REPO_REMOTE_PATH}/pkg/" "${REPO}/pkg"
 
-ssh ${remote_macos_host} rm -rf "${TMP_WORKSPACE}"
+ssh ${remote_macos_host} chmod -R +w "${TMP_WORKSPACE}"
```

I'd add `|| true` here to suppress errors and continue with build here too.

```suggestion
ssh ${remote_macos_host} chmod -R +w "${TMP_WORKSPACE}" || true
```